public/locales/en-US/translations.json

```diff
@@ -17,6 +22,8 @@
   "xrpl_explorer": "XRPL Explorer",
   "ledgers": "Ledgers",
   "ledger": "Ledger",
+  "taxon": "Taxon",
+  "token_id": "Token Id",
```

Comment:
This should probably be called `NFTokenId` to match the rippled format

Reply:
This value is used for the header and trying to keep it as plain of a name as possible.  For raw output we will use the actual field name.  This is also what we call it on the NFTokenMint transaction display.